pkg/overlay/cache.go

```diff
@@ -108,15 +108,19 @@ type NodeDossier struct {
 
 // NodeStats contains statistics about a node.
 type NodeStats struct {
-	Latency90          int64
-	AuditSuccessRatio  float64
-	AuditSuccessCount  int64
-	AuditCount         int64
-	UptimeRatio        float64
-	UptimeSuccessCount int64
-	UptimeCount        int64
-	LastContactSuccess time.Time
-	LastContactFailure time.Time
+	Latency90             int64
+	AuditSuccessRatio     float64
+	AuditSuccessCount     int64
+	AuditCount            int64
+	UptimeRatio           float64
+	UptimeSuccessCount    int64
+	UptimeCount           int64
+	LastContactSuccess    time.Time
+	LastContactFailure    time.Time
+	AuditReputationAlpha  float64
```

Comment:
Is this change supposed to part of this PR?

Reply:
Yes.  It probably shouldn't be, but this tiny change is why we started examining this code in the first place.  https://github.com/storj/storj/pull/2206 was the original scope, but we realize that nobody actually used the protocol buffer version of NodeStats